setup.py

```diff
@@ -1,30 +1,32 @@
-from setuptools import setup, find_namespace_packages
+# `pragma: no cover` excludes code from coverage.py
+from setuptools import setup, find_namespace_packages  # pragma: no cover
 
-install_requires = ["numpy"]
-tests_require = ["pytest", "pytest-cov", "pytest-flake8"]
-dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
-scm_version_template = """# Generated by setuptools_scm
-__all__ = ["__version__"]
+if True:  # pragma: no cover
```

Comment:
@parejkoj found a way to do it and I will adopt it. For now I'm leaving this commit because I don't want to mess up the review by rebasing, but I have pushed a `.coveragerc` file that renders this ugly change to `setup.py` unnecessary and I look forward to dropping it.